go.mod

```diff
@@ -53,6 +53,8 @@ require (
 )
 
 require (
+	github.com/Delta456/box-cli-maker/v2 v2.3.0
```

Comment:
Are there packages cross platform compatible? Especially for windows?

Reply:
So `box-cli-maker` seems to render fine in windows command line.  `go-term-markdown` is another story...

Reply:
For windows me might have to revert back to just showing a table (not sure how tables even look with really long comments today).  I'm not aware of any `go+markdown+windows-cli` packages.

Reply:
We can maybe look into this...https://github.com/charmbracelet/glow